test/mobile/test_lite_script_type.py

```diff
@@ -0,0 +1,43 @@
+import torch
+import torch.utils.bundled_inputs
+import io
+from typing import NamedTuple
+
+from torch.jit.mobile import _load_for_lite_interpreter
+from torch.testing._internal.common_utils import TestCase, run_tests
+
+
+class TestLiteScriptModule(TestCase):
+
+    def test_namedtuple_custom_classtype(self):
+        class Foo(NamedTuple):
+            id: torch.Tensor
```

Comment:
Could you add more fields in this test, such as  `name: foo`, etc., for a more generic case?

Reply:
How about just a NamedTuple, without a class wrapper? Could you also add it to the test?